app/becomeSitter/layout.tsx

```diff
@@ -0,0 +1,9 @@
+const BecomeSitterLayout = ({children}: any) => {
+    return (
+        <>
+            { children }
+        </>
+    ) 
+}
+
+export default BecomeSitterLayout;
```

Comment:
what do we use those for?

Reply:
that's the new page for listing all the owners, pretty much the same as the findSitters one but this one shows all the owners listed instead of all the sitters listes like it is in findSitter